front/app/component-library/package.json

```diff
@@ -0,0 +1,13 @@
+{
+    
+  "name": "@citizenlab/cl2-component-library",
```

Comment:
This is a special component library package json that is only used for publishing to npm.